cooker/bin/commands/start/request-listener.js

```diff
@@ -336,17 +335,21 @@ async function requestListener(req, res) {
           file = mdFile;
         }
         // try the global locale for the page
-        locale = path.resolve(
-          config.PAGES_DIRECTORY,
-          potentialLocale,
-          pagePath.join("/"),
-          config.OPTIONS.locales.directoryName,
-          config.OPTIONS.locales.defaultName + ".yaml"
-        );
-
-        if (!fse.pathExistsSync(locale)) locale = null;
-      } else {
-        file = path.join(pageLocalePath, "..", pageName.replace("html", "pug"));
+        const defaultLocalesObj = await resolveLocales(pagePath, config.OPTIONS.locales.defaultName);
+        locales = defaultLocalesObj.locales;
+        // LIAM 2021-08-14 - I'm commenting this out because I want to review it later on.
```

Comment:
`potentialLocale` comes from line `311` which splits the directory path.
I named it `potentialLocale` because if it's a nested path it might not be a locale.

Reply:
This basically does what you are doing inside `resolveLocales`

Reply:
I think it's safe to delete everything, now that you have that function, it already deals with this in a more elegant and functional way.